asyncprawcore/codes.py

```diff
@@ -1,4 +1,4 @@
-"""Use our own codes instead of requiring Requests."""
+"""Use our own codes instead of requiring Requests/Aiohttp."""
```

Comment:
Requests isn't used at all, so I think you don't need to include it at all. 

```suggestion
"""Use our own codes instead of requiring Aiohttp."""
```